tests/ui_suite/fav_test.py

```diff
@@ -83,8 +103,7 @@ def test_add2fav_1_2_1(logined_page):
 
 
 # HELPERS
+@allure.step('Comparing actual favorite list with expected')
```

Comment:
yep, that's what I meant - that is a good use of allure method annotations, and it's much better than multiple marking the same steps inside tests.